configure.in

```diff
@@ -2793,8 +2786,8 @@ fi
 # error should be last step after all autoconf checks are performed, otherwise
 # false side-effects happens.
 if test "$GCC" = yes -a "$ICC" = no; then
-  PGAC_PROG_CC_CFLAGS_OPT([-Werror=uninitialized])
-  PGAC_PROG_CC_CFLAGS_OPT([-Werror=implicit-function-declaration])
+   PGAC_PROG_CC_CFLAGS_OPT([-Wall])
```

Comment:
The -Wall flag has already been set here https://github.com/arenadata/gpdb/blob/15315c3cf96988e37868e9dbac1294f41ebfda7c/configure.in#L518